powerstore/resource_host.go

```diff
@@ -92,6 +91,28 @@ func (r *resourceHost) Schema(ctx context.Context, req resource.SchemaRequest, r
 								string(gopowerstore.InitiatorProtocolTypeEnumFC),
 							)},
 						},
+						"chap_mutual_password": schema.StringAttribute{
+							Description:         "Password for CHAP authentication. This value must be 12 to 64 UTF-8 characters. This password cannot be queried. CHAP password is required when the cluster CHAP mode is mutual authentication.",
+							MarkdownDescription: "Password for CHAP authentication. This value must be 12 to 64 UTF-8 characters. This password cannot be queried. CHAP password is required when the cluster CHAP mode is mutual authentication.",
```

Comment:
Do we need validation for providing chap_mutual_username and chap_mutual_password together?

Reply:
As discussed, validation is present from API side for length.